orte/mca/ess/pmi/ess_pmi_module.c

```diff
@@ -161,17 +161,7 @@ static int rte_init(void)
         error = "getting univ size";
         goto error;
     }
-    orte_process_info.max_procs = u32;
-
```

Comment:
It looks like this is reverting the PMIX_UNIV_SIZE commit from earlier, was that intended?